tutorials/platform/html5_shell_classref.rst

```diff
@@ -0,0 +1,266 @@
+
+    Start the instance of the engine, using the passed strings as
+    command line arguments. :js:meth:`engine.startGame` can be used 
+    in typical cases instead.
+    
+    This will initialize the instance, if it is not initialized. For manual 
+    initialization see :js:meth:`engine.init`. The engine must be loaded beforehand.
+
+    Fails, if a canvas cannot be found on the page.
+
+    :param string variadic:
+        Command line argument.
+
+    :returns:
+        Promise that resolves once the engine started.
+
+.. js:method:: engine.startGame(execName, mainPack)
+
+    Start the game instance using given executable URL and main pack URL.
+
+    This will initialize the instance, if it is not initialized. For manual 
+    initialization see :js:meth:`engine.init`.
+
+    This will load the engine, if it is not loaded. The base path of the 
+    executable URL will be used as the engine base path.
+
+    :param string execName:
+        Executable name in a form of URL, omitting filename extension.
+
+    :param string mainPack:
+        URL of the main pack to start the game.
+
+    :returns:
+        Promise that resolves once the game started.
+
+.. js:method:: engine.setUnloadAfterInit(enabled)
+
+    Specify whether the engine will be unloaded automatically after the
+    instance is initialized. Enabled by default.
+
+    :param boolean enabled:
+        ``true`` if the engine shall be unloaded after initializing,
+        ``false`` otherwise.
+
+.. js:method:: engine.setCanvas(canvasElem)
+
+    Specify a canvas HTML element to use. By default, the first canvas element
+    on the page is used for rendering.
+
+    :param HTMLCanvasElement canvasElem:
+        The canvas element to use.
+
+.. js:method:: engine.setCanvasResizedOnStart(enabled)
+
+    Specifies whether the canvas will be resized to the width and height
+    specified in the project settings on start. Enabled by default.
+
+    :param boolean enabled:
+        ``true`` if the canvas shall be resized on start, ``false`` otherwise.
+
+.. js:method:: engine.setLocale(locale)
+
+    Specify a language code to select the proper localization for the game.
+    
+    .. seealso:: Complete list of :ref:`supported locales <doc_locales>`.
+
+    :param string locale:
+        Language code.
+
+.. js:method:: engine.setExecutableName(execName)
+
+    Specify the virtual filename of the executable. By default, the base name 
+    of the loaded engine files is used.
+
+    This affects the output of :ref:`OS.get_executable_path() <class_OS_method_get_executable_path>`
+    and sets the automatically started main pack to :file:`{ExecutableName}.pck`.
+
+    :param string execName:
+        Executable name.
+
+.. js:method:: engine.setProgressFunc(callback)
+
+    Specify a callback function for displaying download progress. Callback function is
+    called once per frame, so that usage of ``requestAnimationFrame()`` is not necessary.
```

Comment:
```suggestion
    Specify a callback function for displaying download progress. The callback function is
    called once per frame, so that usage of ``requestAnimationFrame()`` is not necessary.
```